docs/application/dotnet/guides/nui/grid-layout.md

```diff
@@ -0,0 +1,29 @@
+# Grid Layout
```

Comment:
Global comment:

Kindly note to update the headings with Sentence Case and Note style as suggested in the styleguide:

- Headings: https://github.com/Samsung/tizen-docs#note
- Note style: https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md